components/common/Text/TextBody.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+import { TextProps } from './TextProps';
+
+const Text = styled.span`
+    font-style: normal;
+    font-weight: normal;
+    font-size: 16px;
+    line-height: 22px;
+    color: ${({ theme, color }) => color ? color : theme.text.defaultColor};
```

Comment:
You can make base `Text` component in order to move the default color logic to it and reuse the component in all the Text components.

Reply:
Made base Text component. Moved color property to it